### DISCUSSION TITLE
Allow the weekly target display to be a rolling 7 days instead of a calendar week

### DISCUSSION THREAD

**Discussion by LivInTheLookingGlass** [iSoron/uhabits#1029] [Feature Requests]
Pretty much the title. I feel like this gives a better idea of whether you need to perform an action on a given day. For instance, if you have 150 minutes of exercise weekly as a goal, it is more useful to know that you have 120 minutes in the last 7 days than to know you have 60 minutes so far this calendar week

**Comment by yancyknight** [comment]
I would love if this was a per habit setting, but if you can only do one or the other I also think the rolling total would be much more useful than the current calendar total. 
I would also want this to apply to all the bars on the Target graph regardless of the goal interval.
